### ISSUE TITLE
strictTemplates, Angular 9, Type 'string' is not assignable to type 'NgbPanel'

### ISSUE THREAD

**Issue by kknD7J3fPUGDMNR** [ng-bootstrap/ng-bootstrap#3619]
### Bug description:
Enabling the new `strictTemplates` flag in Angular 9's compiler options generates the following build error:

`error TS2322: Type 'string' is not assignable to type 'NgbPanel'`

pointing to any use of `ngbPanelToggle` i.e.

```
    <ng-template ngbPanelHeader let-opened="opened">
      <div class="d-flex align-items-center justify-content-between">
        <h5 class="m-0">First panel - {{ opened ? 'opened' : 'collapsed' }}</h5>
        <button ngbPanelToggle class="btn btn-link p-0">Toggle first</button>
      </div>
    </ng-template>
```
(taken from https://ng-bootstrap.github.io/#/components/accordion/examples#header)

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 9.0.2

ng-bootstrap: 6.0.0

Bootstrap: 4.4.1

**Comment by maxokorokov** [comment]
@kknD7J3fPUGDMNR unfortunately at the moment I'd suggest you use the combination of:

```ts
"strictTemplates": true,
"strictInputTypes": false,
```

to disable the input type checks to "workaround" the problem. At the moment our public APIs don't fully conform to `strictTemplates`, but might in future.

**Comment by peterblazejewicz** [comment]
> to disable the input type checks to "workaround" the problem. At the moment our public APIs don't fully conform to `strictTemplates`, but might in future.

Can we have a list of things to change/overhaul in separate item, so we could iterate on them? Thx!

**Comment by maxokorokov** [comment]
@peterblazejewicz actually looking into it right now, not so many things to change. Will open a PR soon.

**Comment by andzejsw** [comment]
I have this error after update:
`This likely means that the library (@ng-bootstrap/ng-bootstrap) which declares NgbModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.`

**Comment by maxokorokov** [comment]
@aleeeftw what update? please open a new issue following the template if you have any problems.

**Comment by aleeeftw** [comment]
@maxokorokov Hmmm?!

**Comment by maxokorokov** [comment]
sorry, @aleeeftw my bad, wrong mention. I meant @andzejsw :)

**Comment by andzejsw** [comment]
Same error for me appears in `@ng-bootstrap/ng-bootstrap` version `6.0.0`. Is it merged in Master or still it isn't fixed?

**Comment by maxokorokov** [comment]
@andzejsw not yet merged, please either use master or wait for `6.0.1`

**Comment by jkruse14** [comment]
@maxokorokov - forgive my ignorance, but how do I install the master branch? When I do `npm install ng-bootstrap/ng-bootstrap#master` it tells me to use yarn. When I try `yarn add` it says I do not have access. I know I must be missing something obvious
